transforms/angle-brackets/transform.js

```diff
@@ -35,7 +35,9 @@ function capitalizedTagName(tagname) {
   return tagname
     .split('-')
     .map(s => {
-      return s[0].toUpperCase() + s.slice(1);
+      let firstChar = s[0];
+      firstChar = s[0] === undefined ? '-' : s[0];
```

Comment:
I don't understand this. Why would `s[0]` be undefined? and why would that control the `-`?

Reply:
I'm simply putting back the iphen if its the first character.

Reply:
I have simplified the logic

Reply:
```js
if (!s) return '-';
return s[0].toUpperCase() + s.slice(1);
```
how about this?

Reply:
how about
```ts
      let firstChar = s[0] || '-';
      return firstChar.toUpperCase() + s.slice(1);
```

Reply:
I can do either LOL